### DISCUSSION TITLE
Can you get how many items are in a collection in total?

### DISCUSSION THREAD

**Discussion by MathiewMay** [pocketbase/pocketbase#2171] [Q&A]
I was wondering if its possible to get a nubmer (1,2,3,4, etc...) of how many iteams are in a collection without, not in a page, in the entire collection.

**Comment by ganigeorgiev** [comment]
The list action (aka. the `getList()` call) returns also the total number of items within the collection that satisfies the `getList()` filter (if any).

Alternatively, if you need more advanced aggregations, you can create a [View collection](https://pocketbase.io/docs/collections/#view-collection) with a plain SQL statement, for example:

```sql
SELECT
  (ROW_NUMBER() OVER()) as id,
  COUNT(*) as totalItems
FROM yourCollection
GROUP BY something
```
